### PR TITLE
Updated Event framework postman collection - Invalid FSP and Block Transfer

### PR DESCRIPTION
**Pull request by amarmodus**
Updated Event framework - Invalid FSP and Block Transfer
*Also updated the common test query for kibana search

### CODE REVIEW

**Comment by amarmodus** [comment]
Updated Event framework test collection
* fulfill abort and time out scenarios added

**Comment by elnyry** [comment]
> Note that the tests for Event Framework should be configurable (i.e. support being enabled or disabled). I do not currently see any support for such a configuration.
> 
> The reason for this is that one wants to be able to run the QA framework against an environment that does not have EFK deployed (i.e. local docker-compose, etc).
> 
> Let me know your thoughts on this.

@mdebarros We have this currently as a separate collection, so that it can be run selectively.. Do you think thats good enough?

**Comment by mdebarros** [comment]
> > Note that the tests for Event Framework should be configurable (i.e. support being enabled or disabled). I do not currently see any support for such a configuration.
> > The reason for this is that one wants to be able to run the QA framework against an environment that does not have EFK deployed (i.e. local docker-compose, etc).
> > Let me know your thoughts on this.
> 
> @mdebarros We have this currently as a separate collection, so that it can be run selectively.. Do you think thats good enough?

Yeah, thats perfect.

File: Event-Framework-Tests.postman_collection.json [hunk 1]
```diff
@@ -246,6 +246,859 @@
...
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"id": "9d58aca2-4bdc-4c4b-b12b-b70c76f3c36d",
+						"exec": [
+							"var uuid = require('uuid');",
+							"var generatedUUID = uuid.v4();",
+							"",
+							"pm.environment.set('transfer_ID', generatedUUID);",
```
Comment (mdebarros) [inline Event-Framework-Tests.postman_collection.json]: Should these not be set in the env file?

File: Event-Framework-Tests.postman_collection.json [hunk 2]
```diff
@@ -246,6 +246,859 @@
...
+							"",
+							"pm.variables.set('transferDate', (new Date()).toUTCString());",
+							"",
+							"pm.environment.set(\"transferExpiration\",new Date(new Date().getTime() + 600000));",
+							"",
+							"pm.variables.set('efk_url', 'http://dev1-elasticsearch.mojaloop.live/_all/_search?pretty');",
```
Comment (mdebarros) [inline Event-Framework-Tests.postman_collection.json]: The Efk URI should be generalised.

File: Event-Framework-Tests.postman_collection.json (hunk 1 above)
Comment (amarmodus) [reply Event-Framework-Tests.postman_collection.json]: uuid, generatedUUID and transfer_ID - always used this way in the gold path postman collection. Efk URI can be moved to env file.

File: Event-Framework-Tests.postman_collection.json (hunk 2 above)
Comment (amarmodus) [reply Event-Framework-Tests.postman_collection.json]: EFK URI is generalised in local env file and other ENV files are updated accordingly.

**Pull request merged**